### PR TITLE
Prepare KEB for CN region

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- prepare a values.yaml with minimal settings to deploy KEB to CN region

**Related issue(s)**

### CODE REVIEW

**Comment by KsaweryZietara** [comment]
Bump KEB to 1.8.2

File: resources/cn/values.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+includeAdditionalParamsInSchema: true
+
+# must be configured
+kubeconfig:
+    allowOrigins: https://dashboard.kyma.cloud.sap
+    clientID: client-id-to-set
+    issuerURL: https://kyma.accounts.ondemand.com
+
+kymaVersion: 2.20.0
+kymaVersionOnDemand: false
```
Comment (KsaweryZietara) [inline resources/cn/values.yaml:58]: `kymaVersionOnDemand` can be removed after bump to 1.8.2

File: resources/keb/templates/deprovision-retrigger-job.yaml [hunk 2]
```diff
@@ -118,3 +123,4 @@ spec:
                 optional: true
           {{- end}}
   schedule: "{{ .Values.deprovisionRetrigger.schedule }}"
+{{ end }}
```
Comment (ralikio) [inline resources/keb/templates/deprovision-retrigger-job.yaml:126]: Missing new line.

File: resources/keb/templates/migrator-job.yaml [hunk 3]
```diff
@@ -55,7 +59,7 @@ spec:
           {{- end }}
         {{- end}}
         - name: migrator
-          image: {{ .Values.global.images.container_registry.path }}/{{ .Values.global.images.schema_migrator.dir }}/schema-migrator:{{ .Values.global.images.schema_migrator.version }}
+          image: {{ .Values.global.images.container_registry.path }}/{{ .Values.global.images.schema_migrator.dir }}schema-migrator:{{ .Values.global.images.schema_migrator.version }}
```
Comment (ralikio) [inline resources/keb/templates/migrator-job.yaml:62]: Values `schema_migrator.dir` does not have trailing slash. What is the reason for that change? If it is needed values.yaml should be changed accordingly.

File: resources/keb/values.yaml [hunk 4]
```diff
@@ -120,7 +120,6 @@ broker:
     memory: false
   events:
     enabled: false
-  enableShootAndSeedSameRegion: "false"
```
Comment (ralikio) [inline resources/keb/values.yaml:123]: I think this line might got removed accidentally, right?

File: resources/keb/templates/deprovision-retrigger-job.yaml [hunk 5]
```diff
@@ -1,3 +1,4 @@
+{{ if .Values.deprovisionRetrigger.enabled }}
```
[the remark this answers is not in the document]
Comment (piotrmiskiewicz) [reply resources/keb/templates/deprovision-retrigger-job.yaml:1]: maybe I'm missing something: https://github.com/kyma-project/kyma-environment-broker/pull/850/files#diff-766f429183c7f1aee18fdd8a07a6a25508a990d7deab24dc97263f448f886575R97

File: resources/keb/templates/migrator-job.yaml (hunk 3 above)
Comment (piotrmiskiewicz) [reply resources/keb/templates/migrator-job.yaml:62]: You are right. The problem is that the "dir" for CN must be empty. I've modified the main values yaml for that

File: resources/keb/values.yaml (hunk 4 above)
Comment (piotrmiskiewicz) [reply resources/keb/values.yaml]: you are right, fixed

File: resources/keb/templates/deprovision-retrigger-job.yaml (hunk 5 above)
[the remark this answers is not in the document]
Comment (ralikio) [reply resources/keb/templates/deprovision-retrigger-job.yaml:1]: My mistake.

**Review by ralikio — APPROVED** [review]

File: internal/process/provisioning/create_runtime_for_own_cluster_step_test.go [hunk 6]
```diff
@@ -173,7 +173,8 @@ func fixConfigMap(defaultKymaVersion string) k8sruntime.Object {
 			},
 		},
 		Data: map[string]string{
-			"default": `additional-components:
+			"default": `kyma-template: "---"
+additional-components:
```
Comment (ralikio) [inline internal/process/provisioning/create_runtime_for_own_cluster_step_test.go:177]: Should KymaTemplate contain additional-components?

**Review by ralikio — APPROVED** [review]

**Review by ralikio — DISMISSED** [review]

File: docs/README.md [hunk 7]
```diff
@@ -42,4 +42,5 @@ For technical details of KEB, go to the `contributor` directory:
 * [Cleaning and Archiving](./contributor/08-10-cleaning-and-archiving.md)
 
 You can also read about:  
-* [Event Data Platform Tools](https://github.com/kyma-project/kyma-environment-broker/blob/main/utils/edp-registrator/README.md)
\ No newline at end of file
+* [Event Data Platform Tools](https://github.com/kyma-project/kyma-environment-broker/blob/main/utils/edp-registrator/README.md
```
Comment (IwonaLanger) [inline docs/README.md:45]:
```suggestion
* [Event Data Platform Tools](https://github.com/kyma-project/kyma-environment-broker/blob/main/utils/edp-registrator/README.md)
```

File: resources/cn/cn.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
+# Installing KEB in CN region
```
Comment (IwonaLanger) [inline resources/cn/cn.md:1]:
```suggestion
# Install KEB in the CN Region
```

File: resources/cn/cn.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+This guide will help you install KEB in the CN region.
+
+## Prerequisites
+
+Push all necessary images to the proper Docker registry.
+Install Istio on the cluster.
```
Comment (IwonaLanger) [inline resources/cn/cn.md:8]:
```suggestion
- All necessary images pushed to the proper Docker registry.
- Istio installed on the cluster.
```

File: resources/cn/cn.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+## Installation
+
+1. Set the proper values in the `sql.yaml`, especially the database password.
+
+2. Prepare a secret with kubeconfig to gardener project:
```
Comment (IwonaLanger) [inline resources/cn/cn.md:14]:
```suggestion
2. Prepare a Secret with a kubeconfig to the Gardener project:
```

File: resources/cn/cn.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+```shell
+KCFG=`cat <file with kubeconfig>`
+kubectl create secret generic gardener-credentials --from-literal=kubeconfig=$KCFG -n kcp-system
+```
+
+3. Prepare a secret with credentials for Docker registry.
```
Comment (IwonaLanger) [inline resources/cn/cn.md:21]:
```suggestion
3. Prepare a Secret with credentials for the Docker registry.
```

**Review by IwonaLanger — APPROVED** [review]

File: docs/README.md [hunk 12]
```diff
@@ -42,4 +42,5 @@ For technical details of KEB, go to the `contributor` directory:
 * [Cleaning and Archiving](./contributor/08-10-cleaning-and-archiving.md)
 
 You can also read about:  
-* [Event Data Platform Tools](https://github.com/kyma-project/kyma-environment-broker/blob/main/utils/edp-registrator/README.md)
\ No newline at end of file
+* [Event Data Platform Tools](https://github.com/kyma-project/kyma-environment-broker/blob/main/utils/edp-registrator/README.md)
+* [Installing KEB in CN region](../resources/cn/cn.md)
```
Comment (IwonaLanger) [inline docs/README.md:46]:
```suggestion
* [Install KEB in the CN Region](../resources/cn/cn.md)
```

**Review by IwonaLanger — APPROVED** [review]

**Review by ralikio — APPROVED** [review]

**Pull request merged**